media/js/firefox/whatsnew/whatsnew-128-experiment-eu.es6.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrafficCop from '@mozmeao/trafficcop';
+import { isApprovedToRun } from '../../base/experiment-utils.es6.js';
+
+const href = window.location.href;
+
+const initTrafficCop = () => {
+    if (href.indexOf('v=') !== -1) {
+        if (href.indexOf('v=1') !== -1) {
+            // UA
+            window.dataLayer.push({
+                'data-ex-variant': 'wnp128-add-ons',
+                'data-ex-name': 'wnp-128-experiment-eu'
+            });
```

Comment:
Old GA was just deactivated last week so I don't think these events work any more. We should switch to the GA4 equivalent (here and for the others):
```suggestion
        // GA4
        window.dataLayer.push({
            event: 'experiment_view',
            id: 'wnp-128-experiment-eu',
            variant: '1'
        });
```

Reply:
I thought we stopped tracking analytics for EU WNPs though?

Reply:
Oh, yeah, good point... so then we can just remove them entirely, right?